### PR TITLE
Show release channels in build dashboard

### PR DESCRIPTION
**Pull request by CaseyHillers**
* The previous way can show the past 60 days worth of branches
* Instead, only show HEAD, stable, beta, dev

### CODE REVIEW

**Review by XilaiZhang — APPROVED** [review]
Thanks for the change Casey!
images (1)

File: app_dart/lib/src/service/branch_service.dart [hunk 1]
```diff
@@ -160,8 +164,8 @@ class BranchService {
       },
       {
         "branch": latestCandidateBranch,
-        "name": "latestCandidateBranch",
-      }
+        "name": "dev",
+      },
```
Comment (godofredoc) [inline app_dart/lib/src/service/branch_service.dart:168]: Aren't we over engineering this solution? why don't we just run a query on the Checklist entity?

Comment (CaseyHillers) [reply app_dart/lib/src/service/branch_service.dart:168]: What type of query do you imagine? We could simplify this with enums to dedupe the code, but it's still going to require lookups to GitHub to find what RC maps to beta and stable. Files from GitHub should be aggressively cached by Cocoon, so this should be a cheap lookup.

**Comment by yusuf-goog** [comment]
@godofredoc @CaseyHillers are the questions resolved?

**Pull request merged**